css/styles.css

```diff
@@ -108,3 +164,209 @@ body {
   display: flex;
   padding: 1px;
 }
+
+.work-section {
+  display: flex;
+  padding-top: 40px;
+  justify-content: center;
+}
+
+.title {
+  font-family: 'Crete Round', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  font-size: 40px;
+  line-height: 52px;
+  margin-left: 24px;
+  border-bottom: 1px solid rgb(0, 0, 0);
+}

```

Comment:
Here the problem is you did't take the good font, you selected the Crete Round italic font, that's why your text is in italic. 
Please go to google font and select the good font
`
@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@1&display=swap");
`